eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -159,7 +159,7 @@ jobs:
           javaHomeOption: 'JDKVersion'
           jdkVersionOption: $(JavaTestVersion)
           jdkArchitectureOption: 'x64'
-          goals: jacoco:report-aggregate
+          goals: org.jacoco:jacoco-maven-plugin:report-aggregate
```

Comment:
The data client builds were encountering an error generating the aggregate report: "No plugin found for prefix 'jacoco'"